app/models/episode.rb

```diff
@@ -132,8 +132,9 @@ def image
     images.first
   end
 
-  def image=(file)
-    img = EpisodeImage.build(file)
+  def image=(img)
+    img = EpisodeImage.new(original_url: img) if img.is_a?(String)
+    img = EpisodeImage.new(img) if img.is_a?(Hash)
```

Comment:
So when this comes from an API PUT, what is the `img` param?  I would have guessed a hash, but sounds like maybe you saw something else?

Reply:
It came in as an `EpisodeImage` instance already